libraries/utilities.js

```diff
@@ -280,24 +290,34 @@ async function getTargetIdFromConfigFile(filePath) {
         return null;
     }
 
-    let resultId = null;
+    return await queryXMLContents(contents, (xml) => {
```

Comment:
If queryXMLContents throws due to the parse failing, this function will bubble the error up. I think the intended behavior is to wrap this `return` in a `try {} catch {}` that returns null if something explodes

Reply:
Thank you for `catch`ing my last `try`, so to speak. Added that.

Reply:
Ah yes, `finally`